girder/cumulus/server/models/cluster.py

```diff
@@ -215,7 +215,7 @@ def mongo_safe_value(value):
             else:
                 for (k, v) in value.iteritems():
                     if '.' in str(k) or '$' in str(k):
-                        k = k.replace('.', '_').replace('$', '_')
+                        k = k.replace('.', u'_').replace('$', u'_')
 
```

Comment:
Sorry I didn't make myself clear. What I meant, was the following:
k = k.replace('.', '\\u002e').replace('$', '\\u0024')